Weather.css

```diff
@@ -117,4 +117,13 @@ form {
 .box:hover {
         width: 400px;
  }
-    
\ No newline at end of file
+
+#errorname {
+    box-sizing: border-box;
+    background:rgba(194, 62, 62, 0.747);
+    font-size: 16px;
+    font-weight: bold;
+    color: black
```

Comment:
Given the background color, maybe white would look better than black here?

Reply:
sure David, we will check on this. Thanks